pipeline/packer/scr-web-service.json

```diff
@@ -0,0 +1,75 @@
+{
+  "_description": "This template produces a docker container for the SCR web service and pushes the result to the reository.",
+  "variables": {
+    "app_dir": "/usr/src/app",
+    "common_dir": "common",
+    "scr_dir": "SCR",
+    "scr_service_dir": "SCRWebService",
+    "repository": "{{env `SCR_REPOSITORY`}}",
+    "tag": "{{env `BUILD_TAG`}}",
+    "docker_registry_url": "{{env `SCR_REPOSITORY_URL`}}"
```

Comment:
I've found locally with other packer files that setting both `SCR_REPOSITORY` and `SCR_REPOSITORY_URL` to the Docker repo url without any https:// part works. Though I haven't checked that it always uses https. But it may be possible to just have one environment variable?